### PR TITLE
GH Actions: update deprecated pio platform command

### PR DESCRIPTION
**Pull request by rickard-von-essen**
These will be removed in the next release.

### CODE REVIEW

**Review by pkendall64 — DISMISSED** [review]

**Review by pkendall64 — COMMENTED** [review]
Dies not seem to be working!

**Comment by rickard-von-essen** [comment]
Let me check what happened.

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -105,9 +105,9 @@ jobs:
 
     - name: Run PlatformIO
       run: |
-        platformio platform update
-        platformio platform install native
         mkdir -p ~/artifacts/firmware
+        platformio pkg install --platform native
+        platformio pkg update
         cd src
```
Comment (pkendall64) [inline .github/workflows/build.yml:111]: Also move this line up as well

**Comment by rickard-von-essen** [comment]
Now it builds for me here https://github.com/rickard-von-essen/ExpressLRS/actions/runs/3796550527
(Can't say how this worked before)

**Comment by pkendall64** [comment]
> Now it builds for me here https://github.com/rickard-von-essen/ExpressLRS/actions/runs/3796550527 (Can't say how this worked before)

I believe that previously it did not need to use the `platformio.ini` file as the installation was global.

**Comment by rickard-von-essen** [comment]
Ok, so maybe instead of moving these lines around we could add the `--global`.

**Comment by pkendall64** [comment]
> Ok, so maybe instead of moving these lines around we could add the `--global`.

Not sure it matters, I think the way it is now is good.

**Review by pkendall64 — APPROVED** [review]

**Review by AlessandroAU — APPROVED** [review]

**Pull request merged**